vars/odsOrchestrationPipeline.groovy

```diff
@@ -88,7 +88,7 @@ def call(Map config) {
 
 private withPodTemplate(String odsImageTag, Closure block) {
     def podLabel = "mro-jenkins-agent-${env.BUILD_NUMBER}"
-    def odsNamespace = env.ODS_NAMESPACE ?: 'ods'
+    def odsNamespace = env.ODS_NAMESPACE ?: 'cd'
```

Comment:
This change looks wrong to me? It should default to `ods`.

Reply:
yes - this is to make my local setup fly / wo modifiying jenkins :) - I'll fix this